### DISCUSSION TITLE
Opening commit message in float window

### DISCUSSION THREAD

**Discussion by Maritsu** [tpope/vim-fugitive#2142] [Q&A] [answered]
Is it possible to change the behavior of `:G commit` to open the commit message buffer in a float window instead of splitting the current window?

**Comment by tpope** [comment, marked as answer]
Not currently possible. Adding this would require opening a can of worms; the APIs are relatively low-level level and require us to muck around with positioning, etc.

**Reply by Maritsu** [reply]
Ah, alright. Thanks
